Sources/KituraContracts/ClousureAliases.swift

```diff
@@ -71,7 +95,7 @@ public typealias IdentifierCodableClosure<Id: Identifier, I: Codable, O: Codable
 The `CodableClosure` is for use in cases where you'd want to perform a series of actions utilising an object conforming to `Identifier` then respond with an object which conforms to `Codable`, which is of the same type as the object passed as a parameter, or responding with a `RequestError` in the form of a `CodableResultClosure`
```

Comment:
Please try rewording as above.

Reply:
I think the first paragraph here should be reworded as we did with the previous alias definition.

Spell check the whole set of files for 'occured' -> 'occurred' and 'appropiate' -> 'appropriate'

The second paragraph reword as follows -
"By default `Int` has conformity to `Identifier`. In this example, if there has been an error you can use the respondWith call to respond with an appropriate error, passing nil for the `User?`. If no errors occurred and you have a `User`, you can just respond with the user, passing nil as the `RequestError?` value."